tests/Agent/IntegrationTests/UnboundedIntegrationTests/MySql/MySqlStoredProcedureTests.cs

```diff
@@ -39,11 +42,6 @@ protected MySqlStoredProcedureTestsBase(MySqlBasicMvcFixture fixture, ITestOutpu
                     CommonUtils.ModifyOrCreateXmlAttributeInNewRelicConfig(configPath, new[] { "configuration", "transactionTracer" }, "recordSql", "raw");
                     CommonUtils.ModifyOrCreateXmlAttributeInNewRelicConfig(configPath, new[] { "configuration", "transactionTracer" }, "explainThreshold", "1");
                     CommonUtils.ModifyOrCreateXmlAttributeInNewRelicConfig(configPath, new[] { "configuration", "datastoreTracer", "queryParameters" }, "enabled", "true");
-                },
-                exerciseApplication: () =>
-                {
-                    _fixture.GetMySqlParameterizedStoredProcedure(paramsWithAtSigns);
-                    _fixture.AgentLog.WaitForLogLine(AgentLogBase.TransactionTransformCompletedLogLineRegex);
```

Comment:
Is there a reason we no longer need/want to wait for this log line?

Reply:
Discussed without Alex and it sounds like without better existing context for why this would be needed, we can assume it's unnecessary since this test passes reliably without.

Reply:
Ok, as we've seen, reliability can't be gauged by running it locally a few times or a few CI passes... Let's try removing it though :)